index.js

```diff
@@ -1,137 +1,63 @@
+    <div class="book-name">${books[i].name}</div>
+    <div class="book-author">${books[i].author}</div>
+    <button class="button" onclick="Book.removeBook(${i})">Remove</button>
+    </div>`;
+    }
   }
 }
 
-function getStoreFormData(key = 'Library-data') {
-  return JSON.parse(localStorage.getItem(key));
-}
-
-function updateStoreFormData(formObj, key = 'Library-data') {
-  localStorage.setItem(
-    key,
-    JSON.stringify(formObj),
-  );
-}
-
-function initBookStorage() {
-  if (!storageAvailable('localStorage')) return;
-  library = Object.assign(library, getStoreFormData());
-}
-
-// BOOK ELEMENT
-
-const listOfBooksElement = document.querySelector('#books .list-books');
-
-function deleteBook(parentContainer, id) {
-  parentContainer.remove();
-  library.remove(id);
-  updateStoreFormData(library);
-}
-
-function CreateBookItemHTML(id, title, author) {
-  const divContainer = document.createElement('div');
-  const bokkTitle = document.createElement('p');
-  const bookAuthor = document.createElement('p');
-  const deleteBookBtn = document.createElement('button');
-  const hr = document.createElement('hr');
-
-  divContainer.id = `Book-${id}`;
-
-  bokkTitle.innerText = title;
-  bokkTitle.classList.add('book-title');
-  bookAuthor.innerText = author;
-  bokkTitle.classList.add('book-author');
-
-  deleteBookBtn.innerText = 'Remove';
-
-  deleteBookBtn.addEventListener('click', () => {
-    deleteBook(divContainer, id);
-  });
-
-  divContainer.appendChild(bokkTitle);
-  divContainer.appendChild(bookAuthor);
-  divContainer.appendChild(deleteBookBtn);
-  divContainer.appendChild(hr);
-
-  return divContainer;
-}
-
-function AddBookToContainerElement(book) {
-  listOfBooksElement.appendChild(CreateBookItemHTML(book.id, book.title, book.author));
-}
-
-function createBookListing() {
-  library.books.forEach((book) => {
-    AddBookToContainerElement(book);
-  });
-}
-
-// ADD book from
+// Create Necessary Functions
 
-const addBookForm = document.querySelector('#book-form');
-const bookTitleInput = addBookForm.querySelector('#title');
-const bookAuthorInput = addBookForm.querySelector('#author');
+const storedBooks = JSON.parse(localStorage.getItem('books'));
 
-function addBook(e) {
-  e.preventDefault();
-  AddBookToContainerElement(library.add(bookTitleInput.value, bookAuthorInput.value));
-  updateStoreFormData(library);
-  return false;
+if(storedBooks) {
+  books.push(...storedBooks);
+  Book.refreshBook();
 }
 
-function addBookButtonLIstener() {
-  addBookForm.addEventListener('submit', addBook);
-}
-
-// INITS
-
-function init() {
-  initBookStorage();
-  if (library) createBookListing();
-  addBookButtonLIstener();
-}
+// Create Event Lisener
+const addBook = document.querySelector('.submit');
+addBook.addEventListener('click', () => {
+  const libro = new Book(bookName.value, bookAuthor.value);
+  books.push(libro);
+  Book.loadBooks(books.length - 1);
+  localStorage.setItem('books', JSON.stringify(books));
+});
```

Comment:
- [ ] Your class should have methods to `add` and `remove` books but this `addBook`` method is not found inside your class. Kindly address this issue :100: